packages/codegen/src/codegen/.hygen/templates/powerhouse/generate-drive-editor/components/Files.esm.t

```diff
@@ -2,7 +2,7 @@
 to: "<%= rootDir %>/<%= h.changeCase.param(name) %>/components/Files.tsx"
 unless_exists: true
 ---
-import { FileItem } from "@powerhousedao/design-system";
+import { FileItem } from "@powerhousedao/design-system/connect";
```

Comment:
these changed import paths are the only thing that will affect existing projects. I wanted to move these files to avoid this change, but doing so would end up resulting in even more changes